pngcanvas.py

```diff
@@ -258,14 +268,16 @@ def load(self, f):
             raise ValueError('Unsupported PNG format (filter_type={}; must be 0)'.format(filter_type))
         if interlace != 0:
             raise ValueError('Unsupported PNG format (interlace={}; must be 0)'.format(interlace))
-        if color_type != 6:
-            raise ValueError('Unsupported PNG format (color_type={}; must be 6)'.format(color_type))
+        if color_type not in (2, 6):
```

Comment:
I'd do the following:

    SUPPORTED_FORMATS = (
        2,  # RGB
        6,  # RGBA
    )
    
    ...
    
    if color_type not in SUPPORTED_FORMATS: